api/utils.py

```diff
@@ -77,3 +79,15 @@ def reestructure_areas(config_dict):
             config_dict[f"Area_{index}"] = config_dict[area_name]
             config_dict.pop(area_name)
     return config_dict
+
+
+def clean_up_file(filename):
```

Comment:
How long does this take to execute? Can the file be deleted before its download is finished?